sdk/core/core-tracing/review/core-tracing.api.md

```diff
@@ -22,7 +22,9 @@ export interface ContextAPI {
 // @public
 export function createSpanFunction(args: CreateSpanFunctionArgs): <T extends {
     tracingOptions?: OperationTracingOptions | undefined;
-}>(operationName: string, operationOptions: T | undefined) => {
+} & {
```

Comment:
Usage example:

without customizing the span:

```ts
// without customization
const span = createSpan("deleteConfigurationSetting", operationOptions);
// setting spanKind to SERVER on the newly created span
const span = createSpan("deleteConfigurationSetting", {...operationOptions, spanOptions: { spanKind: SpanKind.SERVER } });
```

Alternatives I considered:

```ts
// separate param to `createSpan`
const span = createSpan("deleteConfigurationSetting", operationOptions, { spanKind: SpanKind.CLIENT });

// nested spanOptions inside tracingOptions:
const createSpanOptions = {
    ...operationOptions
    tracingOptions: {
        ...operationOptions.tracingOptions,
        spanOptions: { kind: SpanKind.SERVER }
    }
}
```

Reply:
meh I guess this could just be `{ tracingOptions?: OperationTracingOptions; spanOptions?: SpanOptions }`